src/styles/global.scss

```diff
@@ -7,3 +7,17 @@
 @import "./variables.scss";
 
 @import "./layout.scss";
+
+:root body {
+  --primary: black;
+  --tertiary: white;
```

Comment:
we have variables.scss file for that. / Why on body?

Reply:
Will replace, it was to check if it will work, since everything got affected